### DISCUSSION TITLE
Team Members Benifits

### DISCUSSION THREAD

**Discussion by kolumnin** [community/community#135614] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How am I excluded from all Teams of GutHub?
Github:"You don’t belong to any teams yet!"

**Comment by erencanbas** [comment]
**Pending Invitation**: If you were recently invited to an organization, you may not have been added to any teams yet. You might need to check your email or GitHub notifications for any pending invitations or team assignments.

**Membership Type:** If you are an **"Outside Collaborator"** rather than an organization member, you may not be added to teams. Outside collaborators typically have access only to specific repositories rather than full organizational access.
